test/unstable-keyword/config/configInCmd.chpl

```diff
@@ -0,0 +1,21 @@
+config var dummy = "dummy"; // to make sure that there's a section for this
+                            // module in --help
+
+@unstable("configVar is unstable.")
+config var configVar = "default";
+
+@unstable("configConst is unstable.")
+config var configConst = "default";
+
+@unstable("configParam is unstable.")
+config var configParam = "default";
```

Comment:
Should this be declared param?

Reply:
I actually dropped this, because:

- `param`s need separate handling at compile time
- we can do that, but it maybe a bit redundant -- if the symbol is used anywhere, we already get a warning. Should we get another one while setting it at command line?

Reply:
I definitely think it's reasonable to not handle in this test.  I would probably still handle it, though